aws.go

```diff
@@ -137,6 +166,11 @@ func (a *AWS) Upload(local, remote string) error {
 			shouldCopy = true
 		}
 
+		if !shouldCopy && contentEncoding != "" {
```

Comment:
with this condition every single file will always be reuploaded if you have a content encoding set